sim/humanoid_gym/play_h5.py

```diff
@@ -0,0 +1,213 @@
+import argparse
```

Comment:
Please create a separate PR for model an keep only the data logic here.

Reply:
https://github.com/kscalelabs/sim/issues/51

i made one here, i'll clean up this pr